tortoise/contrib/pydantic/creator.py

```diff
@@ -131,6 +131,7 @@ def pydantic_model_creator(
     exclude_readonly: bool = False,
     meta_override: Optional[Type] = None,
     model_config: Optional[ConfigDict] = None,
+    __validators__: dict[str, Any] | None = None,
```

Comment:
IMHO this should be received as `validators` (without the underscores at beginning and end).